### PR TITLE
More reasonable and restrained cuts to Section 3 of the introduction

### PR DESCRIPTION
**Pull request by pavpanchekha**
This section is pretty powerful so I didn't want to make any big changes that could hurt that. But in a few places I thought it could be shortened.

One big question I have about this section is the last paragraph. I think this paragraph is what my kind of dopey trailer in #55 was trying to be. What do you think of moving this section ("The browser and me") before the previous one ("Explaining the black box")? And then using the last paragraph of this section as the last paragraph of "Explaining the black box"?

### CODE REVIEW

File: book/browsersandweb.md [hunk 1]
```diff
@@ -163,98 +162,88 @@ been far from it in my studies or work.
...
-complicated set of optimizations deemed necessary to squeeze out the last bit of
-performance from the system. And there is the painstaking, but necessary, work
-to continuously refactor the code to reduce its complexity through the
+[^meantime-linux]: Meanwhile, the "better Linux browser than Netscape" took a
+long time to appear....
+
+How hard indeed! After seven years in the trenches working on Chrome, I now know
+the answer to his question: building a browser is both easy and incredibly hard,
+both intentional and accidental, both planned and organic, both simple and
+unimaginably complex. Everywhere you look, you see the evolution and history of
```
Comment (chrishtr) [inline book/browsersandweb.md:191]:
```suggestion
unimaginably complex. And everywhere you look, you see the evolution and history of
```

File: book/browsersandweb.md [hunk 2]
```diff
@@ -163,98 +162,88 @@ been far from it in my studies or work.
...
 on other operating systems.
 
-
-[^meantime-linux]: Meanwhile, the “better Linux browser than Netscape” seemed to
-take quite a long time to appear....
-
-How hard indeed! After seven years in the trenches working on a browser
-(Chrome), I now know the answer to his question: building such a browser is
-both easy and incredibly hard, intentional and accidental, planned and organic,
-simple and unimaginably complex. And everywhere you look, you can see the
```
Comment (chrishtr) [inline book/browsersandweb.md:196]: I think I like retaining "and" here.

File: book/browsersandweb.md [hunk 3]
```diff
@@ -163,98 +162,88 @@ been far from it in my studies or work.
...
+
+These kinds of quandaries are common to all complex codebases. But what makes a
+browser different is that there is often an _urgency to fix them_. Browsers are
+nearly as old as any “legacy” codebase, but are _not_ legacy, not abondoned or
+half-deprecated, not slated for replacement. On the contrary, they are vital to
+the world’s economy. Browser engineers are forced[^forced-negative] to fix and
+improve rather than abaondon and replace.
+
+[^forced-negative]: I don't intend the negative connotation. It’s because
+browsers are dynamic important that we browser developers can afford an
```
Comment (chrishtr) [inline book/browsersandweb.md:245]:
```suggestion
browsers are so important that we browser developers can afford an
```

**Comment by pavpanchekha** [comment]
You're right, the "and" works well. I'll let you merge this if you like it.

**Review by chrishtr — APPROVED** [review]

**Pull request merged**